crates/aggchain-proof-service/src/custom_chain_data/mod.rs

```diff
@@ -6,12 +6,26 @@ use serde::{Deserialize, Serialize};
 #[cfg(test)]
 mod tests;
 
-type VKeySelector = [u8; 4];
+#[derive(Debug)]
+// TODO: Making this unused as it will be used in another iteration
+#[allow(unused)]
+pub struct VKeySelector([u8; 4]);
```

Comment:
Does `#[expect()]` work here? That way we'll get heads up from the compiler if it becomes used but it does not always work with all possible build options (test vs non-test, various features enabled, etc).

```suggestion
#[expect(unused)]
pub struct VKeySelector([u8; 4]);
```

Reply:
After trying, it doesn't work well I think.
I will merge it as is